package.json

```diff
@@ -9,7 +9,7 @@
   ],
   "author": "SAP SE (https://www.sap.com)",
   "license": "SEE LICENSE IN LICENSE",
-  "homepage": "https://cap.cloud.sap/",
+  "homepage": "https://github.com/cap-js/cds-adapter-graphql",
```

Comment:
This should rather be the [`repository.url`](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#repository).
NPM will display this link as _Repository_.

Let's reserve the `homepage` for some documentation page, I'd say.

Reply:
changed to repository.url and reverted homepage